### PR TITLE
Use component logger on Paper for console colors

### LINKED ISSUE #4895: Paper console colors are broken
### Type of bug

Compatibility issue, Other unexpected behaviour

### `/ess dump all` output

N/A

### Error log (if applicable)

_No response_

### Bug description

Paper changed the logger to not parse color codes and must now use the console command sender for colors in the console.

### Steps to reproduce

Do any command that gives feedback messages in the console.

### Expected behaviour

Doing /echest without permission for example does
[Essentials] §cHexedHero was denied access to command.
As you can see the console is now a mess with random §'s without any color.

### Actual behaviour

It should be
[Essentials] HexedHero was denied access to command.
while being colored red

### PR DESCRIPTION
**Pull request by JRoy**
Fixes #4895

### CODE REVIEW

**Review by mdcfe — DISMISSED** [review]
I'm really not a fan of replacing every single usage of a logger with a static method.

File: Essentials/src/main/java/com/earth2me/essentials/EssentialsUpgrade.java [hunk 1]
```diff
@@ -722,7 +720,7 @@ private void deleteOldItemsCsv() {
                 doneFile.setProperty("deleteOldItemsCsv", true);
                 doneFile.save();
             } catch (final IOException ex) {
-                Bukkit.getLogger().log(Level.SEVERE, ex.getMessage(), ex);
+                Essentials.getWrappedLogger().log(Level.SEVERE, ex.getMessage(), ex);
```
Comment (JRoy) [inline Essentials/src/main/java/com/earth2me/essentials/EssentialsUpgrade.java]: why

File: Essentials/src/main/java/com/earth2me/essentials/EssentialsUpgrade.java [hunk 2]
```diff
@@ -748,7 +746,7 @@ private void updateSpawnsToNewSpawnsConfig() {
                     config.blockingSave();
                 }
             } catch (final Exception ex) {
-                Bukkit.getLogger().log(Level.SEVERE, ex.getMessage(), ex);
+                Essentials.getWrappedLogger().log(Level.SEVERE, ex.getMessage(), ex);
```
Comment (JRoy) [inline Essentials/src/main/java/com/earth2me/essentials/EssentialsUpgrade.java]: why

File: Essentials/src/main/java/com/earth2me/essentials/EssentialsUpgrade.java [hunk 3]
```diff
@@ -776,7 +774,7 @@ private void updateJailsToNewJailsConfig() {
...
```
Comment (JRoy) [inline Essentials/src/main/java/com/earth2me/essentials/EssentialsUpgrade.java]: why

File: Essentials/src/main/java/com/earth2me/essentials/I18n.java [hunk 4]
```diff
@@ -89,7 +88,6 @@ private String translate(final String string) {
             }
         } catch (final MissingResourceException ex) {
             if (ess == null || ess.getSettings().isDebug()) {
-                Logger.getLogger("Essentials").log(Level.WARNING, String.format("Missing translation key \"%s\" in translation file %s", ex.getKey(), localeBundle.getLocale().toString()), ex);
```
Comment (JRoy) [inline Essentials/src/main/java/com/earth2me/essentials/I18n.java:92]: why

File: Essentials/src/main/java/com/earth2me/essentials/commands/Commandgc.java [hunk 5]
```diff
@@ -57,7 +57,7 @@ protected void run(final Server server, final CommandSource sender, final String
                     tileEntities += chunk.getTileEntities().length;
                 }
             } catch (final java.lang.ClassCastException ex) {
-                Bukkit.getLogger().log(Level.SEVERE, "Corrupted chunk data on world " + w, ex);
+                Essentials.getWrappedLogger().log(Level.SEVERE, "Corrupted chunk data on world " + w, ex);
```
Comment (JRoy) [inline Essentials/src/main/java/com/earth2me/essentials/commands/Commandgc.java]: we have `ess` here

File: Essentials/src/main/java/com/earth2me/essentials/items/FlatItemDb.java [hunk 6]
```diff
@@ -115,7 +113,7 @@ public ItemStack get(String id, final boolean useResolvers) throws Exception {
         }
 
         id = id.toLowerCase();
-        final String[] split = id.split(":");
+        final String[] split = id.split("(?<!^minecraft):");
```
Comment (JRoy) [inline Essentials/src/main/java/com/earth2me/essentials/items/FlatItemDb.java]: somebody got the branches out of sync

File: Essentials/src/main/java/com/earth2me/essentials/I18n.java (hunk 4 above)
Comment (mdcfe) [reply Essentials/src/main/java/com/earth2me/essentials/I18n.java:92]: no idea what happened here

**Review by mdcfe — APPROVED** [review]

**Pull request merged**